### PR TITLE
NXP-29263: handle login links with anonymous user

### PR DESCRIPTION
**Pull request by atchertchian**
T&P at https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-atchertchian-master-2/99/

### CODE REVIEW

File: modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-webengine/src/main/resources/skin/nxlogin.ftl [hunk 1]
```diff
@@ -1,88 +1,6 @@
...
-      <input type="submit" name="nuxeo_login" value="Ok" id="login" class="button">
-    </form>
--->
+<#if Context.principal.isAnonymous()>
+  <a href="/nuxeo/login.jsp?forceAnonymousLogin=true&requestedUrl=${Root.path}" id="login">Login</a>
 <#else>
-  <div class="logout"><span id="logstate">${Context.principal.name}</span>
-  <a href="" id="logout">Logout</a></div>
+  <span id="logstate">${Context.principal.name}</span>
+  <a href="/nuxeo/logout" id="logout">Logout</a>
```
Comment (efge) [inline modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-webengine/src/main/resources/skin/nxlogin.ftl:5]: I'm not sure hardcoding `/nuxeo` is a good idea. I see other FTL files using `Context.serverURL` or doing `<@extends src="base.ftl">` then using `basePath` or `Root.path`. I'm not sure I'm not sure of the exact details but it's worth checking.

**Comment by atchertchian** [comment]
new T&P https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-atchertchian-master-2/100/

**Comment by atchertchian** [comment]
T&P passed ok :heavy_check_mark:

**Comment by atchertchian** [comment]
Removing last commit https://github.com/nuxeo/nuxeo/pull/4169/commits/7a0415852883184c519f13cd3ac614e3151e34cf from PR, will be handled with https://jira.nuxeo.com/browse/NXP-29203 instead

**Pull request merged**